README.md

```diff
@@ -2,69 +2,58 @@
-
-> Please also take into account that Kubeapps is using the [Bitnami chart](https://github.com/bitnami/charts/tree/master/bitnami/kubeapps) as the official one. The chart inside this repository is only intended to be used for development purposes.
+> Kubeapps is using the [Bitnami chart](https://github.com/bitnami/charts/tree/master/bitnami/kubeapps) as the official one. The chart inside this repository is only intended to be used for development purposes.
 
-## Developer Documentation
+## Documentation
 
 Please refer to:
 
-- The [Kubeapps Build Guide](./docs/developer/build.md) for instructions on setting up the build environment and building Kubeapps from source.
-- The [Kubeapps Developer Documentation](./docs/developer/README.md) for instructions on setting up the developer environment for developing on Kubeapps and its components.
-
-## Next Steps
-
-If you have followed the instructions for [installing Kubeapps](./docs/user/getting-started.md) check how to [use Kubeapps](./docs/user/dashboard.md) to easily manage your applications running in your cluster, or [look under the hood to see what's included in Kubeapps](./docs/architecture/overview.md).
-
-## Useful Resources
-
-- [Walkthrough for first-time users](./docs/user/getting-started.md)
+- [Getting started guide](./docs/user/getting-started.md)
 - [Detailed installation instructions](chart/kubeapps/README.md)
-- [Kubeapps Dashboard documentation](./docs/user/dashboard.md)
-- [Kubeapps components](./docs/architecture/overview.md)
-- [Project board](https://github.com/kubeapps/kubeapps/projects/11)
-- [Roadmap](https://github.com/kubeapps/kubeapps/milestones)
-
-## API documentation
-
-Whereas Kubeapps does not have yet a well-designed API, we still want to share the current status of the APIs used across the internal components. Check out the [latest API online documentation](https://app.swaggerhub.com/apis/kubeapps/Kubeapps) or download the Kubeapps [OpenAPI Specification yaml file](./dashboard/public/openapi.yaml) from the repository.
-
-Feel free to contribute using this guide on [how to update the API docs](./docs/developer/update-api-docs.md) and send us a PR if you find any errors.
-
-> This API documentation is subject to change in each Kubeapps version. If you are interested in the Kubeapps API, please [reach out to us](https://kubernetes.slack.com/messages/kubeapps) for further information.
+- [Kubeapps user guide](./docs/user/dashboard.md) to easily manage your applications running in your cluster.
+- [Kubeapps build guide](./docs/developer/build.md) for setting up the build environment and building Kubeapps from source.
+- [Kubeapps developer guide](./docs/developer/README.md) for setting up the developer environment.
+- [Kubeapps architecture](./docs/architecture/overview.md)
```

Comment:
I'd rather remove them from this section and will wait until we have a chance to revisit them. Later, I'd add these links in a `## Contributing` section (along with the contributing.md, and other files)